client/src/components/ImportAssignmentButton.tsx

```diff
@@ -77,6 +77,18 @@ const ImportAssignmentButton = (props: ImportAssignmentButtonProps) => {
           onUpload={onUpload}
           importing={importing}
           onImport={onImport}
+          acceptedTypes={[
```

Comment:
I think this should not be a prop. This will be the same for all imports, right?

Reply:
No, the answers should still allow the upload of non-archive files.

Reply:
Okay, but for archive uploads, the different kinds of archives are always the same.

Reply:
Yes.

Reply:
Then we should at least extract this list into a constant so that we don't have to maintain in in multiple places.